src/components/molecules/ToggleTabs/ToggleTabs.tsx

```diff
@@ -23,21 +25,36 @@ const ToggleTabs: FC<ToggleTabsProps> = ({
   setActiveTab,
   tabs = [],
   hidden,
+  dateTabsClassName,
+  value,
```

Comment:
**[blocker]**

A bit confused about these 2 props (value and onChange), they seem to just duplicate the exact behaviour of activeTab and setActiveTab without doing anything extra. If you just want to rename the original ones, so you could us the tabs as a Form Input component, then let's remove the previous names for these props rather than duplicate them